cpp/mcap/include/mcap/mcap.hpp

```diff
@@ -108,4 +127,78 @@ struct UnknownRecord {
   mcap::ByteArray data;
 };
 
+struct McapWriterOptions {
+  bool indexed;
```

Comment:
```suggestion
  bool indexed = false;
```
I generally put initializers on all primitives to help avoid accidental use of uninitialized data

Reply:
The initializer is a few lines down

Reply:
I seem to remember some strangeness where it was possible to get uninitialized data even if there was a user defined constructor, but can't come up with it now... maybe the convention was established just to avoid forgetting to initialize members when changing the constructor definitions (which may be far away in a different file)

Reply:
https://abseil.io/tips/61

> They will help reduce bugs from omissions, especially when someone adds a new constructor or a new data member.